### PR TITLE
Feat: this adds the app's setup

### PR DESCRIPTION
**Pull request by ezehlivinus**
I setup the server,
I installed express and **reactJS package**

### CODE REVIEW

**Comment by Superior212** [comment]
@ezehlivinus Kindly trigger the function in the index.js file  @startupApp

File: app.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const startApp = () => {
+  console.info('App is starting...');
+}
```
Comment (ezehlivinus) [inline app.js]: @ezehlivinus please trigger the....

**Comment by ezehlivinus** [comment]
> @ezehlivinus Kindly trigger the function in the index.js file @startupApp

Okay, I will do that right push again. Thanks @Superior212

**Comment by Superior212** [comment]
> > @ezehlivinus Kindly trigger the function in the index.js file @startupApp
> 
> Okay, I will do that right push again. Thanks @Superior212

Alright.

**Review by ezehlivinus — COMMENTED** [review]
a comment

**Pull request merged**